docs/operate/get-started/other-hardware/_index.md

```diff
@@ -177,7 +177,11 @@ Edit the generated files to add your logic:
 
 1. **Edit the `reconfigure` function** to do the following:
 
-   - Assign any default values as necessary to any optional attributes if the user hasn't configured them.<br><br>
+   - Get any values from the `config` object that the user has configured.
+   - Assign default values as necessary to any optional attributes if the user hasn't configured them.
+   - If your module has dependencies, get the dependencies from the `dependencies` map and cast each resource according to which API it implements, as in [this <file>ackermann.py</file> example](https://github.com/mcvella/viam-ackermann-base/blob/main/src/ackermann.py).
```

Comment:
Maybe show it once instead?
```
from typing import cast

        motors = config.attributes.fields["drive_motors"].list_value
        for m in motors:
            actual_motor = dependencies[Motor.get_resource_name(m)]
            motor = cast(Motor, actual_motor)
```

Reply:
Tried this in staging and I sort of think it takes up too much space and doesn't have enough context around it, esp. since not everyone will have dependencies.